tests/e2e/pageObjects/rdi-instances-list-page.ts

```diff
@@ -73,6 +73,23 @@ export class RdiInstancesListPage extends BaseOverviewPage {
         }
     }
 
+    /**
+     * Edit Rdi by name
+     * @param dbName The name of the rdi to be deleted
+     */
+    async editRdiByName(dbName: string): Promise<void> {
```

Comment:
but it is the same as the method below? clickOnEditRdiByName

Reply:
yes, thank you